lib/fastlane/plugin/revenuecat_internal/actions/bump_phc_version_create_pr_action.rb

```diff
@@ -0,0 +1,109 @@
+require 'fastlane/action'
+require 'fastlane_core/configuration/config_item'
+require_relative '../helper/revenuecat_internal_helper'
+require_relative '../helper/versioning_helper'
+
+module Fastlane
+  module Actions
+    class BumpPhcVersionCreatePrAction < Action
```

Comment:
Since creating the PR is an optional operation, should we rename this action to `BumpPhcVersion` only?